cassandra-launcher/src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala

```diff
@@ -107,7 +108,7 @@ object CassandraLauncher {
         case jarUrl if jarUrl.getProtocol == "jar" =>
           new File(URI.create(jarUrl.getPath.takeWhile(_ != '!'))).getCanonicalPath
       }
-    }.distinct.to[immutable.Seq]
+    }.distinct.to(immutable.Seq)
```

Comment:
This is the only place where the added `collectionCompat` is used. We could instead do `.distinct.toIndexedSeq` and the collectionCompat would not be needed.

Reply:
I'll have a look

Reply:
TIL `IndexedSeq` in Scala 2.12 is `scala.collection.IndexedSeq` but `toIndexedSeq` returns `scala.collection.immutable.IndexedSeq`.